developer-docs/bolt_server.md

```diff
@@ -0,0 +1,145 @@
+# Bolt Server 
+
+## Overview
+Bolt server provides a service to run bolt tasks over SSH and WinRM. This service is exposed via an API as described in this document. 
```

Comment:
I think it's worth adding a line along the lines of "Bolt server works as a standalone service, and is available in PE Johnson and greater as `pe-bollt-server`".